eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -25,12 +25,13 @@ public static function dumpAssets(CommandEvent $event)
         $options = self::getOptions($event);
         $appDir = $options['symfony-app-dir'];
         $webDir = $options['symfony-web-dir'];
-        $env = isset($options['ezpublish-asset-dump-env']) ? $options['ezpublish-asset-dump-env'] : '';
+        // @deprecated: Use of ezpublish-asset-dump-env is deprecated, use standard SYMFONY_ENV instead.
```

Comment:
Proposal: Lets then get rid of the question, and add `--env` parameter only if value is set in `ezpublish-asset-dump-env` *(bc)*, and otherwise not set it so command uses `SYMFONY_ENV` and if not falls back to `dev`.

thumbs up or down reaction on this comment?? @bdunogier @gggeek @emodric